docs/adr/005-Use-HTML-sanitizer-to-avoid-XSS-attacks.md

```diff
@@ -0,0 +1,83 @@
+# 005. Use HTML sanitizer to avoid XSS attacks
+
+Date: 29-no-2021
+
+## Status
+
+PROPOSED  
+
+## Summary
+
+*CKEditor4 team detected an XSS vulnerability on the Mathtype devkit package. A user could inject malicious to a website code encaplusating them inside a <math> tag.*
+
+*In order to prevent XSS attacks it is proposed to use the DOMPurif library. An open-source XSS sanitizer for HTML, MathML and SVG.*
+
+## Context
+
+Browser engine *Parsing* is the step the browser takes to turn the data it receives over the network into the DOM. Given the following block, which uses the `<math>` tag:
+
+```html
+<math>
+  <style>
+    <img src onerror="alert(1)" />
+  </style
+</math>
+```
+
+The browser egine will generate it to: 
+
+```html
+<MathML math>
+  <MathML style>
+    <MathML img src onerror="alert(1)" />
+```
+
+Normally and `<img>` inside  an `<style>` will be parsed as a `#text` inside the `<MathML style>` tag. Nevertheless, this behaviour change when is encapsulated inside a `<math>` tag, and interprets the img as a `<MathML>` independent block. As result the HTML will be unwrapped and writen on the DOM following this format:
+
+```html
+<math>
+  <img src onerror="alert(1)"/>
+  <style></style>
+</math>
+```
+
+This parsing behaivor can be used to inject malicious JavaScript on a website. In order to avoid this vulnerability is needed to filter all the HTML added from the editor.
+
+Implementing a sanitize system can be laborious and will be difficult to maintenance in order to detect as much XSS vulnerabilities as possible.
+
+## Decision
+
+DOMPurify sanitize HTML and prevents XSS attacks. You can feed DOMPurify with string full of dirty HTML and it will return a string removing the dangerous HTML.
+
+DOMPurify will be added to `@wiris/mathtype-html-integration-devkit` so this dependency will affect to all the `html-integrations` packages.
+
+### Pros and Cons of the Options 
+
+#### Implement our own Javascript library for that
+
+- Bad, because it is a huge task to detect all of possible XSS injections.
```

Comment:
I thing that there's a good part on it, since we would have full control on the library. By adding this good clause, i don't mean that the option to create our own library is better, but it's good to have documented all pros and cons

Reply:
Added the good part.